lib/design/app_theme.dart

```diff
@@ -0,0 +1,61 @@
+
+import 'package:flutter/material.dart';
+
+import 'my_colors.dart';
+
+class AppTheme {
+
+  final BorderRadius buttonBorderRadius = BorderRadius.circular(18.0);
+  final EdgeInsets mainButtonPadding = EdgeInsets.all(15);
+
+  ThemeData get themedata {
+    return ThemeData(
+        backgroundColor: MyColors.white,
+        buttonTheme: ButtonThemeData(
+          buttonColor: MyColors.darkGreen,
+          padding: mainButtonPadding,
+          height: 50,
+          minWidth: 200,
+          //* I strongly recommend that we continue using the .primary attribute
+          //* since this will automatically detect the best color for the text
```

Comment:
Nice. Add (johanehinger) att the back of the string so we know who thought this.
Finish the sentence with dot.